block/manager.go

```diff
@@ -117,10 +116,17 @@ func NewManager(
 	if err != nil {
 		return nil, err
 	}
+
+	// TODO(mtsitrin): Probably should be validated and manage default on config init
 	// TODO(omritoptix): Think about the default batchSize and default DABlockTime proper location.
 	if conf.DABlockTime == 0 {
-		logger.Info("WARNING: using default DA block time", "DABlockTime", defaultDABlockTime)
-		conf.DABlockTime = defaultDABlockTime
+		logger.Info("WARNING: using default DA block time", "DABlockTime", config.DefaultNodeConfig.DABlockTime)
+		conf.DABlockTime = config.DefaultNodeConfig.DABlockTime
+	}
+
+	if conf.BlockBatchSizeBytes == 0 {
+		logger.Info("WARNING: using default DA batch size bytes limit", "DABlockTime", config.DefaultNodeConfig.BlockBatchSizeBytes)
```

Comment:
typo. it says "DABlockTime" in the log message (I guess meant to be "BatchSize").